src/ui/MessageInput/index.tsx

```diff
@@ -445,7 +510,18 @@ const MessageInput = React.forwardRef<HTMLInputElement, MessageInputProps>((prop
                 if (!isMobileIOS(navigator.userAgent)) {
                   e.preventDefault();
                 }
-                sendMessage();
+                // fork note: allow enter to edit too
+                if (isEdit) {
+                  editMessage();
```

Comment:
what does this do?

Reply:
does this mean the edit input component is no longer a separate component?

Reply:
The `MessageInput` component within UIKit is used by them for both
- New Message (https://github.com/sendbird/sendbird-uikit-react/blob/main/src/modules/GroupChannel/components/MessageInputWrapper/MessageInputWrapperView.tsx#L196), and
- Editing messages (https://github.com/sendbird/sendbird-uikit-react/blob/main/src/modules/GroupChannel/components/Message/MessageView.tsx#L333).

This particular change is just so that pressing `Enter` (when not holding `shift`) also makes the callback to save the edit. Before, `Enter` would only auto-send on sending new messages, but not when editing.

Reply:
Oh and yea, the plan then is to do the same thing at Gather, and use this same `MessageInput` component for both editing and new message.